platform/features/layout/res/templates/fixed.html

```diff
@@ -25,15 +25,20 @@
     <!-- Background grid -->
     <div class="l-grid-holder" ng-click="controller.clearSelection()">
         <div class="l-grid l-grid-x"
+             ng-if="!controller.getGridSize()[0] < 3"
              ng-style="{ 'background-size': controller.getGridSize() [0] + 'px 100%' }"></div>
         <div class="l-grid l-grid-y"
+             ng-if="!controller.getGridSize()[1] < 3"
              ng-style="{ 'background-size': '100% ' + controller.getGridSize() [1] + 'px' }"></div>
     </div>
 
     <!-- Fixed position elements -->
     <div ng-repeat="element in controller.getElements()"
-         class="l-fixed-position-item"
-         ng-class="{ 's-not-selected': controller.selected() && !controller.selected(element) }"
+         class="l-fixed-position-item s-selectable s-moveable s-hover-border"
+         ng-class="{
+            's-not-selected': controller.selected() && !controller.selected(element),
+            's-selected': controller.selected(element)
```

Comment:
@charlesh88 it seems we could reduce the number of css classes (i.e. remove `.s-not-selected` by implementing `.s-not-selected` in `s-selectable` and have `s-selected` override properties).